modules/core/src/test/java/org/apache/fluo/core/oracle/StampTest.java

```diff
@@ -0,0 +1,34 @@
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.fluo.core.oracle;
+
+import org.junit.Test;
+/**
+ * Unit test for Stamp class
+ */
+public class StampTest {
+	
+	@Test(expected=IllegalArgumentException.class)
+	public void testBadStartTime(){
+	   new Stamp(1<<62, 5);
```

Comment:
My suggestion was slightly wrong.  Need to change `1<<62` to `1L << 62`.  Just the `1` is an int and does not work... adding the `L` makes it a long.

Reply:
@keith-turner  Just made changes accordingly also run mvn compile which shows build successful.